homeassistant/components/huisbaasje/config_flow.py

```diff
@@ -53,9 +53,6 @@ async def async_step_user(self, user_input=None):
         except HuisbaasjeException as exception:
             _LOGGER.warning(exception)
             errors["base"] = "invalid_auth"
-        except Exception:  # pylint: disable=broad-except
-            _LOGGER.exception("Unexpected exception")
-            errors["base"] = "unknown"
```

Comment:
Your commit message and commit do not match.

It is indeed not advised to catch broad exceptions in an entry setup, however, this is NOT the entry setup, this is the config flow.

In the config flow, however, doing this is normal. I would suggest reverting this snippet.

Reply:
I initially removed the code catching all exceptions because it also catches the AbortFlow exception thrown by `_abort_if_unique_id_configured` if the id already exists. I re-added the catch, along with an additional case for the AbortFlow

Reply:
Looks good!
Final remark in this same category/comment:

![image](https://user-images.githubusercontent.com/195327/105357030-80717880-5bf4-11eb-89f3-c5fc0a1dc7b6.png)

Tests are not covering that part of the configuration flow.

Reply:
Fixed, I re-added a test which I removed before